observability/logging/logging-6.0/log6x-about.adoc

```diff
@@ -119,6 +137,113 @@ spec:
       name: logging-loki
 ----
 
+. Create a `ClusterLogForwarder` CR to configure log forwarding:
++
+[source,yaml]
+----
+apiVersion: observability.openshift.io/v1
+kind: ClusterLogForwarder
+metadata:
+  name: collector
+  namespace: openshift-logging
+spec:
+  serviceAccount:
+    name: collector
+  outputs:
+    - name: default-lokistack
+      type: lokiStack
+      lokiStack:
+        target:
+          name: logging-loki
+          namespace: openshift-logging
+        dataModel: ViaQ # <1>
```

Comment:
I'm not sure we should specify `dataModel: ViaQ` in the example, because it will be copied as-is without thinking about it and then keep customers stuck in the old model, even when we eventually deprecate it.

If this would be on a special "try out the new features" page then it would be a different matter, but this is still the first page everyone looking at 6.1 will read, so the "default example" should be as minimal as possible.